pysondb/cli.py

```diff
@@ -203,8 +205,7 @@ def set_parser(argv: Optional[Sequence[str]] = None) -> int:
     convert_to_csv_parser.add_argument(
         "-t",
         "--target_csv",
-        help="path of the converted csv, default : converted.csv",
-        default="converted.csv",
```

Comment:
Why on earth would you remove a default argument, this prevent the argument from being empty.
Thus rendering you PR pointless.
Feels like you are "creating" problems to solve.